app/api/routers/breeze_buddy/merchant_entities/handlers.py

```
@@ -0,0 +1,257 @@
+
+
+def _check_create_merchant_access(current_user: UserInfo):
+    """Check if user can create merchant entities (admin only)."""
+    if current_user.role != UserRole.ADMIN:
+        raise HTTPException(
+            status_code=403, detail="Only admins can create merchant entities"
+        )
+
+
+def _check_update_delete_access(
+    current_user: UserInfo, merchant_entity: MerchantEntityResponse
```

Ruff (0.14.14)

[warning] Unused function argument: `merchant_entity`

(ARG001)